sdk/core/core-sse/test/server/start.mts

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import express from "express";
+import { port, waitBetweenEventsInMS } from "./config.mts";
+import { logger } from "./logger.mts";
+import { sendEvents, sendHeaders } from "./responses.mts";
+
+async function run() {
+  const app = express();
+
+  app.get("/events/:eventCount", async function (req, res) {
+    logger.info(`Got /events/${req.params.eventCount}`);
+    sendHeaders(res);
+    const eventCount = parseInt(req.params.eventCount);
+    await sendEvents(res, eventCount, waitBetweenEventsInMS);
+    res.end();
+  });
+
+  app.get("/events/no-fin/:eventCount", async function (req, res) {
+    logger.info(`Got /events/no-fin/${req.params.eventCount}`);
+    sendHeaders(res);
+    const eventCount = parseInt(req.params.eventCount);
+    await sendEvents(res, eventCount, waitBetweenEventsInMS);
+    res.socket?.destroy();
+    //res.end();
+  });
+
+  app.get("/events/infinite", async function (_, res) {
+    logger.info("Got /events/infinite");
+    sendHeaders(res);
+    await sendEvents(res, 0, waitBetweenEventsInMS);
```

Comment:
Nit, when I read "infinite" here I assumed that we would be sending an infinite number of events. Looks like it's in fact sending zero but is leaving the socket open?

Reply:
Addressed in https://github.com/Azure/azure-sdk-for-js/pull/30707/commits/15035295cc9f51ff77d13c28f13eb57daff12ef8